dask/distributed/worker.py

```diff
@@ -517,6 +523,15 @@ def close(self):
     def __del__(self):
         self.close()
 
+    def heartbeat(self, pulse=5):
+        """Send a message to scheduler at a given interval"""
+        while self.status != 'closed':
+            header = {'function': 'heartbeat'}
+            payload = {}
+            future = self.pool.apply_async(self.send_to_scheduler,
+                                           args=(header, payload))
+            self._heartbeat_thread.event.wait(pulse)
```

Comment:
Ah, I see.  There are other ways.  This one seems simpler than anything else I can think of though.